images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -10,7 +10,15 @@ ENV NGINX_RUN_USER www-data
 ENV NGINX_DOCUMENT_ROOT /home/site/wwwroot
 # Install NGINX 
 RUN apt-get update \
-    && apt-get install nano nginx -y
+    && apt-get install nano curl gnupg2 ca-certificates lsb-release debian-archive-keyring -y \
+    && curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor \
+        | tee /usr/share/keyrings/nginx-archive-keyring.gpg >/dev/null \
+    && gpg --dry-run --quiet --import --import-options import-show /usr/share/keyrings/nginx-archive-keyring.gpg \
+    && echo "deb [signed-by=/usr/share/keyrings/nginx-archive-keyring.gpg] \
+        http://nginx.org/packages/debian `lsb_release -cs` nginx" \
+        | tee /etc/apt/sources.list.d/nginx.list \
+    && apt update \
+    && apt install nginx
```

Comment:
Could you please clarify which NGINX version we'll get by default from this?

Reply:
The default version it installs will be 1.22.